src/main/java/com/dreammaster/gthandler/recipes/ElectrolyzerRecipes.java

```diff
@@ -14,287 +14,486 @@ public class ElectrolyzerRecipes implements Runnable {
-                ItemList.Cell_Empty.get(8L),
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Glauconite, 21L),
+                ItemList.Cell_Empty.get(6L),
                 GT_Values.NF,
                 Materials.Hydrogen.getGas(2000L),
-                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Iron, 3L),
-                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Aluminiumoxide, 4L),
-                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.SiliconDioxide, 4L),
-                GT_OreDictUnificator.get(OrePrefixes.cell, Materials.Water, 4L),
-                GT_OreDictUnificator.get(OrePrefixes.cell, Materials.Oxygen, 4L),
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Potassium, 1L),
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Magnesium, 2L),
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Aluminiumoxide, 10L),
+                GT_OreDictUnificator.get(OrePrefixes.cell, Materials.Oxygen, 6L),
+                GT_Values.NI,
                 GT_Values.NI,
                 null,
-                420,
+                378,
                 120);
+        // Vermiculite
         GT_Values.RA.addElectrolyzerRecipe(
-                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Zeolite, 41L),
-                GT_Utility.getIntegratedCircuit(1),
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Vermiculite, 37L),
+                ItemList.Cell_Empty.get(2L),
                 GT_Values.NF,
-                Materials.Oxygen.getGas(18000L),
-                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Sodium, 1L),
-                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Calcium, 4L),
-                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.SiliconDioxide, 27L),
-                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Aluminiumoxide, 9L),
+                Materials.Hydrogen.getGas(10000L),
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Iron, 3L),
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Aluminiumoxide, 10L),
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.SiliconDioxide, 12L),
+                GT_OreDictUnificator.get(OrePrefixes.cell, Materials.Oxygen, 2L),
                 GT_Values.NI,
                 GT_Values.NI,
                 null,
-                944,
+                420,
                 120);
+        // Mica
         GT_Values.RA.addElectrolyzerRecipe(
-                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Aluminiumoxide, 5L),
-                GT_Utility.getIntegratedCircuit(1),
-                GT_Values.NF,
-                Materials.Oxygen.getGas(3000L),
-                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Aluminium, 2L),
-                GT_Values.NI,
+                GT_OreDictUnificator.get(OrePrefixes.dust, Materials.Mica, 13L),
```

Comment:
2+1+15+9 is not 13

Reply:
Fixed it but unfortunately 1 oxygen has to appear out of nowhere, I can change tooltip in materials.java if necessary

Reply:
maybe alu dust then, so that it can work

Reply:
Mica will output Alumina, but give raw Silicon instead of the silicon dioxide